js/modules/timeline.src.js

```diff
@@ -331,6 +367,8 @@ seriesType('timeline', 'line', {
     }
 });
 
```

Comment:
in the function `getConnectorPath`, set `inverted ? 'x2' : 'y2'`as variable, We will increase readability!